### PR TITLE
Add extra explanation to aid devs in manually setting up tests

### LINKED ISSUE #220: Add clarifying explanations to some of the set up code to allow devs to copy the process
See https://github.com/SuffolkLITLab/docassemble-AssemblyLine-documentation/issues/352.

### PR DESCRIPTION
**Pull request by plocket**
Closes #220

### CODE REVIEW

File: docassemble/ALAutomatedTestingTests/data/questions/al_testing_files_to_push.yml [hunk 1]
```diff
@@ -30,8 +30,10 @@ content: |
     
   % endfor
 ---
-# The closing '}}' doesn't really need to be contained.
-# It just seemed more consistent
+comment: |
+  The setup interview adds this file to the repo during GitHub test setup.
```
Comment (BryceStevenWilley) [inline docassemble/ALAutomatedTestingTests/data/questions/al_testing_files_to_push.yml:34]:
```suggestion
  The setup interview adds this file (run through mako) to the repo during GitHub test setup.
```

Developers can't quite copy-paste this file, due to the large number of `${ '${{' }` in it.

**Review by BryceStevenWilley — APPROVED** [review]
Solid changes! TBH, I'm not sure by themselves they'd close #220 (as in it's still be difficult to copy the process and jump around what this interview is doing), but this is definitely in the right direction of better code comments.

**Comment by plocket** [comment]
Thanks. This is going to be on the advanced concepts page. There might be more documentation on the docs page itself, but I mostly hope they'll understand what they're doing.

File: docassemble/ALAutomatedTestingTests/data/questions/al_testing_files_to_push.yml (hunk 1 above)
Comment (plocket) [reply docassemble/ALAutomatedTestingTests/data/questions/al_testing_files_to_push.yml:34]:
```suggestion
  The setup interview adds this file (run through mako) to the repo during GitHub test setup. If you're creating this file manually, make sure to remove the Mako.
```

**Pull request merged**